src/message_sender.py

```diff
@@ -0,0 +1,44 @@
+from config import Config
+import json
+import requests
+
+
+class MessageSender:
```

Comment:
Regarding naming. What you think about reducing naming redundancy. I am thinking something like:
`Messenger.send()`.

and yes I am well aware I also don't always adhere to this in my own code...